query-engine/prisma-models/src/composite_type.rs

```diff
@@ -22,3 +22,9 @@ impl CompositeType {
         self.fields().find(|f| f.db_name() == db_name)
     }
 }
+
+impl std::fmt::Debug for CompositeType {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_tuple("CompositeType").field(&self.name().to_string()).finish()
```

Comment:
(no need to change anything if it's not the case, but) I think the `to_string()` here can probably be omitted.